docs/nidcpower/examples.rst

```diff
@@ -1,7 +1,9 @@
 Examples
 ========
 
-`You can download all nidcpower examples here <https://github.com/ni/nimi-python/releases/download/1.2.0/nidcpower_examples.zip>`_
+`You can download all nidcpower examples for master version here <https://github.com/ni/nimi-python/tree/master/src/nidcpower/examples>`_
+
+`You can download all nidcpower examples for latest version here <https://github.com/ni/nimi-python/releases/download/1.2.0/nidcpower_examples.zip>`_
```

Comment:
I do like saying "for the laters release".
You could even specify which version:

> You can download all nidcpower 1.2.0 examples here

Reply:
Currently not all the drivers have the same latest version. For such drivers, the latest version does not match the version in examples URL. Explicitly mentioning the version would cause unnecessary confusion.